### DISCUSSION TITLE
Very Simple Source Code Viewing Setup

### DISCUSSION THREAD

**Discussion by cuppajoeman** [doxygen/doxygen#11063] [General]
Hey there, I have a basic program which I use to generate html output so that I can take a codebase and just start browsing and reading files without the need for an editor: 

https://github.com/user-attachments/assets/54917fdd-9476-41fb-b50d-c37d217a2d44

One thing that I love about doxygen are all the links it generates in the source code files which allow you to navigate to declarations and everything like that, I was wondering if it's possible to configure doxygen to get output similar to what shown in my video, thank you!

**Comment by doxygen** [comment]
I'm not sure what you consider similar, but did you try Doxygen with the following settings:

```
EXTRACT_ALL            = YES
RECURSIVE              = YES
GENERATE_HTML          = YES
GENERATE_TREEVIEW      = YES
SOURCE_BROWSER         = YES
```
